scripts/buildhost/start_ec2_instance.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
```

Comment:
may be to use terraform for provision? Easier/more maintainable than a bash script

Reply:
Using terraform to deploy a single machine wouldn't give us that much. I'm not sure the overhead here would justify that. ( maybe I'm talking from position, as I've had lots of technical issues with terraform in the past )

Reply:
I agree with Pavel, these sorts of things should be done with a higher level tool in the future. We use ansible/terraform elsewhere so hopefully this can be rolled up into our other infrastructure at some point. EC2 instances can also be stopped / resumed, rather than completely reprovisioned each time if the goal is cost saving.

Reply:
We need to run a *single* task at a time. Using ansible/terraform for that won't save us much.
Stopping/resuming instances is also an issue. We want to start from a "fresh" state every time.
( just like travis )